contents/handbook/engineering/release-new-version.md

```diff
@@ -38,7 +38,7 @@ The release manager is ultimately responsible for the timeline of the release. T
 
 1. [ ] Post in #dev about the upcoming release (replace `<version>` and `<array draft pr>` from Joe)
 ```
-Release is happening next Wednesday. Which means 
+Release is happening next Monday. Which means 
```

Comment:
I _believe_ we set Wednesday as the release day, because users are more likely to be off on Monday (or Friday) due to holidays (which can have an impact on marketing outreach). Is this correct or am I misremembering something? @joethreepwood

Reply:
Actually, from a marketing perspective I've typically seen that Monday is the best day for deliverability. Users are back from the weekend and start the week by going through emails, etc. 

_I_ thought the reason to do the release on Wednesday was because Engineering didn't want to have to rush things in over the weekend or deal with changes slipping in then.

Reply:
Monday sounds good then! I don't think it's problematic from an engineering perspective either.